manifests/module/director.pp

```diff
@@ -70,6 +70,7 @@
   Integer[1,65535]          $api_port       = 5665,
   Optional[String]          $api_username   = undef,
   Optional[String]          $api_password   = undef,
+  Optional[Boolean]         $service        = false,
```

Comment:
There is a docstring missing for the new parameter.

Reply:
implemented in 158c975